modules/php/Cards/Actions/ActionMoveACreeper.php

```diff
@@ -16,14 +16,18 @@ public function __construct($cardId, $uniqueId)
     );
   }
 
-  public $interactionNeeded = "keeperSelection";
+  public $interactionNeeded = "keeperSelectionOther";
 
```

Comment:
why this change? the card description says "Choose any Creeper in front of any player" - so you can select one of your own of one of any opponent, right?

Reply:
Yes, you're right. To be honest I didn't pay too much attention to it. I simply noticed this wasn't matching the name of the function in JS and did a quick fix to be able to focus on the layout.

Reply:
Probably a merge issue from the creepers branch and the changes in the action resolves.
Originally there was only the "keeperSelection", and we changed that because some actions allow own selection and some only on other players.